Frameworks/UIKit/UIStoryboard.mm

```diff
@@ -51,6 +52,7 @@ + (instancetype)storyboardWithName:(NSString*)name bundle:(NSBundle*)bundle {
         if (storyInfo) {
             ret->_entryPoint = [storyInfo objectForKey:@"UIStoryboardDesignatedEntryPointIdentifier"];
             ret->_fileMap = [storyInfo objectForKey:@"UIViewControllerIdentifiersToNibNames"];
+            ret->_fileMapByStoryboardId = [storyInfo objectForKey:@"UIViewControllerStoryboardIdentifiersToNibNames"];
```

Comment:
The map is stored in the info.plist that we generate in the storyboardc folder, alongside UIViewControllerIdentifiersToNibNames. It is not modifying the nib file output.